web/embed-dart.html

```diff
@@ -55,10 +55,7 @@
   <body class="fullbleed vertical layout" unresolved>
     <div class="flex layout horizontal">
       <div class="relative vertical layout flex-6">
-        <div class="button-bar">
-          <paper-fab id="run-button" icon="av:play-arrow" mini></paper-fab>
-          <paper-fab id="export-pad" icon="launch" mini></paper-fab>
-        </div>
+        <dartpad-buttons></dartpad-buttons>
```

Comment:
As mentioned, it is nicer to encapsulate even if it's a single use instance for readability.